src/dpp/wsclient.cpp

```diff
@@ -52,7 +52,7 @@ void websocket_client::connect()
 		"GET " + this->path + " HTTP/1.1\r\n"
 		"Host: " + this->hostname + "\r\n"
 		"pragma: no-cache\r\n"
-		"User-Agent: DPP/0.1\r\n"
+		"User-Agent: " + "DiscordBot (https://github.com/brainboxdotcc/DPP, 16.0.38)" + "\r\n"
```

Comment:
Why this concatenation here?

Reply:
Placeholder, so it can be replaced later with the actual version detection stuff.

Reply:
please use the actual version number, you should fix the broken version string in queues.cpp, the static one, and expose it as non-static const or constexpr in the dpp namespace, you can then use it here.

The reason the version string there is broken is that the define values contain the version content as hex. You'll need to convert them as hex into text, so 0x10 is version 10:

```cpp
static std::string http_version = "DiscordBot (https://github.com/brainboxdotcc/DPP, " + std::to_string(DPP_VERSION_MAJOR) + "." + std::to_string(DPP_VERSION_MINOR) + "." + std::to_string(DPP_VERSION_PATCH) + ")";
```